pydis_site/apps/api/models/bot/tag.py

```diff
@@ -42,7 +42,7 @@ def validate_tag_embed_fields(fields):
                 validator(value)
 
 
-def validate_tag_embed_footer(footer):
+def validate_tag_embed_footer(footer: Any) -> None:
```

Comment:
Same as for the previous comment, according to the [Discord documentation](https://discordapp.com/developers/docs/resources/channel#embed-object-embed-footer-structure), all items will be mappings of `str => str`. This could be:

```suggestion
def validate_tag_embed_footer(footer: Dict[str, str]) -> None:
```